### PR TITLE
:package: vercel/fiber implementation

### PR DESCRIPTION
**Pull request by jamie-kieran-martin**
Implemented a simple example for adapting fiber to vercel's serverless function golang runtime

https://vercel.com/docs/runtimes#official-runtimes/go

### CODE REVIEW

File: vercel/api/index.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/gofiber/adaptor/v2"
+	"github.com/gofiber/fiber/v2"
+)
+
+func Handler(w http.ResponseWriter, r *http.Request) {
```
[the remark this answers is not in the document]
Comment (ReneWerner87) [reply vercel/api/index.go:10]: please add a comment @jamie-kieran-martin

**Comment by ReneWerner87** [comment]
can you(@jamie-kieran-martin ) please update the example, add a readme and link the example in the root directory in the readme

then i will merge it within a few hours, unfortunately we did not have the pull request in mind

**Comment by jamie-kieran-martin** [comment]
Hi @ReneWerner87, that's all good. This was a fun little example, that I thought someone might find useful, if they were looking to integrate Fiber and Vercel. It might not actually be very useful at all... 😄

**Comment by ReneWerner87** [comment]
ok

**Pull request closed without merging**